src/sentry/api/paginator.py

```diff
@@ -330,3 +330,43 @@ def get_result(self, limit, cursor=None, count_hits=False):
             hits=min(len(self.scores), MAX_HITS_LIMIT) if count_hits else None,
             max_hits=MAX_HITS_LIMIT if count_hits else None,
         )
+
+
+class SnubaOffsetPaginator(object):
```

Comment:
This name could probably more generic since this doesn't actually depend on Snuba specific APIs